src/react-apps/applications/service-development/__tests__/Administration.test.tsx

```diff
@@ -147,6 +154,47 @@ describe('HandleMergeConflictAbort', () => {
     expect(instance.state.serviceDescription).not.toEqual(mockServiceDescription);
   });
 
+  it('should handle sucessfully updating service id', async () => {
+    const wrapper = mount(
+      <AdministrationComponent
+        classes={mockClasses}
+        language={mockLanguage}
+        service={mockService}
+        serviceName={mockServiceName}
+        serviceNameIsSaving={mockServiceNameIsSaving}
+        serviceDescription={mockServiceDescription}
+        serviceDescriptionIsSaving={mockServiceDescriptionIsSaving}
+        initialCommit={mockInitialCommit}
+        serviceId={mockServiceId}
+        serviceIdIsSaving={mockServiceIdIsSaving}
+      />,
+    );
+
+    const instance = wrapper.instance() as AdministrationComponent;
+
+    const mockEvent = {
+      target: {
+        value: 'New id',
+      },
+    };
+
+    expect(instance.state.serviceId).toEqual(mockServiceId);
+    expect(instance.state.editServiceId).toEqual(false);
+    instance.onServiceIdChanged(mockEvent);
```

Comment:
Is it possible to trigger the event through some UI-elements?

Reply:
I have tried before to make onchange event happen before using jest simulate event, but I was uncessful at that time, so havent really looked more into it